modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,53 +3,57 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can have multiple instances in the same environment. 
+In API Manager 2.x, instances of an API can exist in multiple environments. Additionally, you can create multiple instances of an API in the same environment. 
 
 You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+After setting up an API version for management, you repeat the process to create another instance. Attempting to configure the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-To create a API instance:
+== Create an API instance
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+The following procedure describes creating a single API instance from Exchange, and configuring the API instance as an endpoint with proxy:
+
+. In *API Administration*, click *Manage API*. 
+. Select *Manage API From Exchange*.
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+To configure version 1.0 of an example API named `jsonplaceholder`:
 +
 image::manage-exchange-end-task.png[]
 +
-. From *API Name*, select the API (`jsonplaceholder`).
-+
-NOTE: OpenAPI Specification (OAS) 3.0 is currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager. For converting the specifications to OAS 2.0, you must use third-party tools, such as APImatic.
-+
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
-. From *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*. 
-+
-For this example, select *Endpoint With Proxy*.
-. In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown (`+http://api.domain.com+`). 
+. In *API name*, enter `jsonplaceholder`.
+. In *API version*, select version `1.0`.
```

Comment:
Once an asset selected, it will allow the user to select the version of the asset to manage.

Reply:
Fixed.